From 95973e3b8478a9b8928567d18166fe444950a543 Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Thu, 29 Jul 2021 11:12:55 -0700
Subject: [PATCH] Use pyyaml 5.1.1
#
#  Copyright (c) 2015 - 2022, Intel Corporation
#  SPDX-License-Identifier: BSD-3-Clause
#

- To satisfy VEA requirements.

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 scripts/requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/requirements.txt b/scripts/requirements.txt
index bf78713f..f6553d7d 100644
--- a/scripts/requirements.txt
+++ b/scripts/requirements.txt
@@ -7,6 +7,6 @@ psutil>=5.4.8
 pandas>=0.23.0
 tables>=3.4.3
 cffi>=1.6.0
-pyyaml>=5.1.0
+pyyaml==5.1.1
 future>=0.17.1
--
2.26.2

